src/pages/hcp/api-docs/packer/[[...page]].tsx

```diff
@@ -0,0 +1,155 @@
+}
+
+type ApiDocsPageProps = InferGetStaticPropsType<typeof getStaticProps>
+const ApiDocsPage: CustomPageComponent<ApiDocsPageProps> = ({
+	apiPageProps,
+}) => {
+	return (
+		<OpenApiPageContents
+			info={apiPageProps.info}
+			operationCategory={apiPageProps.operationCategory}
+			// Truncate operation paths,
+			// as they are otherwise very difficult to read
+			massageOperationPathFn={(path) =>
+				path.replace(
+					'/packer/2021-04-30/organizations/{location.organization_id}/projects/{location.project_id}',
+					''
+				)
+			}
+			// Add an introductory warning text to each operation
+			// to make a note that the paths have been truncated
+			renderOperationIntro={function PathAside({ data }) {
+				return (
+					<>
+						<DevDotContent>
+							<div className="alert alert-info">
+								<strong>Note:</strong> Operation paths have been truncated for
+								clarity. The full path to this operation is:
+							</div>
+						</DevDotContent>
+						<CodeBlock
+							code={data.__path}
+							theme="dark"
+							options={{ showClipboard: true }}
+						/>
+					</>
+				)
+			}}
+		/>
+	)
+}
+
+export async function getStaticPaths() {
+	let paths = []
+
+	const swaggerFile = await fetchGithubFile(targetFile)
+	const schema = await processSchemaString(swaggerFile)
+
+	if (schema) {
+		paths = getPathsFromSchema(schema)
+	}
+
+	return { paths, fallback: false }
+}
+
+export async function getStaticProps({ params }) {
+	const swaggerFile = await fetchGithubFile(targetFile)
+	const schema = await processSchemaString(swaggerFile)
+
+	// API page data
+	const apiPageProps = getPropsForPage(schema, params)
+
+	// Product data
+	const productData = cachedGetProductData(productSlug)
+
+	// Layout props
+	const headings = []
+
+	// Breadcrumbs
+	const breadcrumbLinks = [
+		{ title: 'Developer', url: '/' },
+		{ title: 'HashiCorp Cloud Platform', url: `/hcp` },
+		{ title: 'API', url: `/hcp/api-docs` },
+	]
+
+	// Breadcrumbs - Render conditional category
+	if ('operationCategory' in apiPageProps) {
+		breadcrumbLinks.push({
+			title: apiPageProps.operationCategory.name,
+			url: `/hcp/api-docs/packer/`,
+		})
+	}
+
+	// Breadcrumbs - Make final element dark-text
+	// @ts-expect-error `isCurrentPage` is an expected optional field
+	breadcrumbLinks[breadcrumbLinks.length - 1].isCurrentPage = true
+
+	// Menu items for the sidebar, from the existing dot-io-oriented navData
+	const apiSidebarMenuItems = apiPageProps.navData.map((menuItem) => {
+		if (menuItem.hasOwnProperty('path')) {
+			// Path differs on dev-dot, so all nodes with `path` must be adjusted
+			return {
+				...menuItem,
+				fullPath: `/hcp/api-docs/packer/${menuItem.path}`,
+			}
+		} else {
+			return menuItem
+		}
+	})
+
+	// Construct sidebar nav data levels
+	const sidebarNavDataLevels = [
+		generateTopLevelSidebarNavData(productData.name),
+		generateProductLandingSidebarNavData(productData),
+		{
```

Comment:
I wonder if we should have the docs sidebar level here as well. It would be more intuitive to me to be able to navigate back to the docs from here.

Reply:
I was hoping to defer the class of UX-specific-feedback to a follow up PR (maybe even post VQA, given that this chunk of work is meant to be scoped to _"stubbing-in"_